### ISSUE TITLE
Firefox version

### ISSUE THREAD

**Issue by Aleck85** [LAJW/history-menu#22]
Do you plan to make version for Firefox? It would be very cool.

**Comment by LAJW** [comment]
Yes, and I had a go at making it work a while ago. But the port is non trivial - there are small differences in the API which make it fail completely. I should try this again.

**Comment by LAJW** [comment]
OK, I gave it a shot again. The good news is that Firefox pretty much supports Manifest V3 now (with a bunch of warnings).

The bad news:
* [Looks like Firefox doesn't provide offline favicons to extensions](https://bugzilla.mozilla.org/show_bug.cgi?id=1315616). We have a similar problem with Opera lagging behind mainline chrome. I could fetch them from Google (as other firefox extensions do), but it's a privacy hazard. I should probably provide an option to turn them off for Firefox.
* The "recent devices" section won't work because [Mozilla decided implementing it is not worth it.](https://bugzilla.mozilla.org/show_bug.cgi?id=1308059#c1)
* Settings sync doesn't seem to work (at least for dev extensions)
